docs/2.develop/contracts/security/random.md

```diff
@@ -0,0 +1,68 @@
+---
+id: random
+title: Random Numbers
+---
+
+When writing smart contracts in NEAR you have access to a `random seed` that enables you to create random numbers/strings
+within your contract.
+
+This `random seed` is **deterministic and verifiable**: it comes from the validator that produced the block signing the previous
+block-hash with their private key.
+
+The way the random seed is created implies two things:
+
+- Only the validator mining the transaction **can predict** what random number will come out. **No one else**.
+Because nobody knows the validators' private key (except the validator itself).
+
+- The validator **cannot interfere** with the random number being created. This is because they need to sign the previous block, over which (with a high probability) they had no control.
+
+
+However, notice that this still leaves room for three types of attacks from the validator:
+1. [Frontrunning](./frontrunning.md), which we cover in another page
+2. Gaming the input
+3. Refusing to mine the block. 
+
+----
+
+## Gaming the Input
+Imagine you have a method that takes an input and gives a reward based on it. For example, you ask the user to choose a number,
+and if it the same as your `random seed` you give them money.
+
+Since the validator knows which `random seed` will come out, it can create a transaction with that specific input and win the price.
```

Comment:
```suggestion
Since the validator knows which `random seed` will come out, it can create a transaction with that specific input and win the prize.
```